core/ops_builtin.rs

```diff
@@ -346,22 +346,23 @@ async fn op_shutdown(
   resource.shutdown().await
 }
 
-#[op]
-fn op_format_file_name(file_name: String) -> String {
-  format_file_name(&file_name)
+#[op2(core)]
+#[string]
+fn op_format_file_name(#[string] file_name: &str) -> String {
+  format_file_name(file_name)
 }
 
 #[op(fast)]
 fn op_is_proxy(value: serde_v8::Value) -> bool {
   value.v8_value.is_proxy()
 }
 
-#[op(v8)]
+#[op2(core)]
 fn op_str_byte_length(
   scope: &mut v8::HandleScope,
-  value: serde_v8::Value,
+  value: v8::Local<v8::Value>,
```

Comment:
I think this can be `&v8::Value`, but I'm wondering if we should just allow this function to throw type errors and make it `fast`. Does it need to handle invalid v8 values?

Reply:
Thinking about it more, it's probably better to keep it slow so we can use utf8_length, but I think `&v8::String` might be better.

Reply:
I expect it will break some API if we change it to `v8::String` (which I assume will make it throw `TypeError` if the passed object is not a string instead of returning `0`).

Reply:
I couldn't see any callers at first glance, but we could revisit this one later on to confirm.

Reply:
I believe it's used in `ext/node` (notice that it's exposed as `core.byteLength()`)